--- grass-7.0.3/include/Make/Shlib.make.orig	2016-04-03 10:07:58.271024747 +0200
+++ grass-7.0.3/include/Make/Shlib.make	2016-04-03 10:27:41.467641758 +0200
@@ -7,7 +7,7 @@
 LDFLAGS += $(SHLIB_LDFLAGS)
 
 $(SHLIB): $(SHLIB_OBJS)
-	$(SHLIB_LD) -o $@ $(LDFLAGS) $^ $(LIBES) $(EXTRA_LIBS) $(MATHLIB)
+	$(SHLIB_LD) -o $@ -Wl,-soname,$(notdir $@) $(LDFLAGS) $^ $(LIBES) $(EXTRA_LIBS) $(MATHLIB)
 ifndef MINGW
 	(cd $(ARCH_LIBDIR); ln -f -s $(notdir $@) $(patsubst %.$(GRASS_VERSION_NUMBER)$(SHLIB_SUFFIX),%$(SHLIB_SUFFIX),$@))
 endif
