--- config.sub.orig	Sat May  3 10:40:25 2003
+++ config.sub	Sat May  3 10:41:36 2003
@@ -317,6 +317,12 @@
 		basic_machine=m68k-apollo
 		os=-bsd
 		;;
+	athlon)
+		basic_machine=athlon -pc
+		;;
+	athlon-*)
+		basic_machine=athlon-`echo $basic_machine | sed 's/^[^-]*-//'`
+		;;
 	aux)
 		basic_machine=m68k-apple
 		os=-aux
@@ -678,7 +684,7 @@
 	pentium | p5 | k5 | k6 | nexen)
 		basic_machine=i586-pc
 		;;
-	pentiumpro | p6 | 6x86 | athlon)
+	pentiumpro | p6 | 6x86)
 		basic_machine=i686-pc
 		;;
 	pentiumii | pentium2)
@@ -687,7 +693,7 @@
 	pentium-* | p5-* | k5-* | k6-* | nexen-*)
 		basic_machine=i586-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
-	pentiumpro-* | p6-* | 6x86-* | athlon-*)
+	pentiumpro-* | p6-* | 6x86-*)
 		basic_machine=i686-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
 	pentiumii-* | pentium2-*)
