diff -Nur grass-6.3.0RC5.orig/lib/gis/Makefile grass-6.3.0RC5/lib/gis/Makefile
--- grass-6.3.0RC5.orig/lib/gis/Makefile	2007-11-23 13:16:02.000000000 +0000
+++ grass-6.3.0RC5/lib/gis/Makefile	2008-02-23 12:57:01.000000000 +0000
@@ -1,7 +1,7 @@
 MODULE_TOPDIR = ../..
 
 LIB_NAME = $(GIS_LIBNAME)
-EXTRA_LIBS = $(XDRLIB) $(DATETIMELIB) $(INTLLIB)
+EXTRA_LIBS = $(XDRLIB) $(DATETIMELIB) $(INTLLIB) $(MATHLIB)
 DATASRC = ellipse.table datum.table datumtransform.table FIPS.code state27 state83 projections gui.tcl
 
 include $(MODULE_TOPDIR)/include/Make/Lib.make
diff -Nur grass-6.3.0RC5.orig/lib/vector/diglib/Makefile grass-6.3.0RC5/lib/vector/diglib/Makefile
--- grass-6.3.0RC5.orig/lib/vector/diglib/Makefile	2007-10-22 20:06:04.000000000 +0100
+++ grass-6.3.0RC5/lib/vector/diglib/Makefile	2008-02-23 12:56:28.000000000 +0000
@@ -1,6 +1,6 @@
 MODULE_TOPDIR = ../../..
 
-EXTRA_LIBS=$(GISLIB) $(RTREELIB)
+EXTRA_LIBS=$(GISLIB) $(RTREELIB) $(MATHLIB)
 
 LIB_NAME = $(DIG2_LIBNAME)
 
diff -Nur grass-6.3.0RC5.orig/lib/vector/rtree/Makefile grass-6.3.0RC5/lib/vector/rtree/Makefile
--- grass-6.3.0RC5.orig/lib/vector/rtree/Makefile	2007-11-18 19:53:18.000000000 +0000
+++ grass-6.3.0RC5/lib/vector/rtree/Makefile	2008-02-23 12:56:28.000000000 +0000
@@ -9,6 +9,8 @@
 	rect.o \
 	split_q.o 
 
+EXTRA_LIBS = $(MATHLIB)
+
 include $(MODULE_TOPDIR)/include/Make/Lib.make
 
 RTLINC = $(ARCH_INCDIR)/rtree
