diff -urNp -x '*.orig' grass-7.6.1.org/lib/python/ctypes/ctypesgencore/parser/cgrammar.py grass-7.6.1/lib/python/ctypes/ctypesgencore/parser/cgrammar.py
--- grass-7.6.1.org/lib/python/ctypes/ctypesgencore/parser/cgrammar.py	2019-03-19 21:05:06.000000000 +0100
+++ grass-7.6.1/lib/python/ctypes/ctypesgencore/parser/cgrammar.py	2021-10-06 23:39:56.737556037 +0200
@@ -1182,7 +1182,7 @@ def p_macro_parameter_list(p):
 
 
 def p_error(t):
-    if t.lexer.in_define:
+    if t.lexer.in_define or t.type == 'PP_END_DEFINE':
         # p_define_error will generate an error message.
         pass
     else:
