diff -Nur grass-6.3.0RC5.orig/include/Make/Shlib.make grass-6.3.0RC5/include/Make/Shlib.make
--- grass-6.3.0RC5.orig/include/Make/Shlib.make	2007-10-20 01:39:46.000000000 +0100
+++ grass-6.3.0RC5/include/Make/Shlib.make	2008-02-23 15:30:58.000000000 +0000
@@ -6,7 +6,7 @@
 LDFLAGS += $(SHLIB_LDFLAGS)
 
 $(SHLIB): $(SHLIB_OBJS)
-	$(SHLIB_LD) -o $@ $(LDFLAGS) $^ $(EXTRA_LIBS) && \
+	$(SHLIB_LD) -o $@ -Wl,-soname,$(notdir $@) $(LDFLAGS) $^ $(EXTRA_LIBS) && \
 	(cd $(ARCH_LIBDIR); ln -f -s $(notdir $@) $(patsubst %.$(GRASS_VERSION_NUMBER)$(SHLIB_SUFFIX),%$(SHLIB_SUFFIX),$@))
 
 shlib: $(SHLIB)
