--- grass-7.0.3/db/drivers/mysql/dbe.c.orig	2012-01-31 22:30:42.000000000 +0100
+++ grass-7.0.3/db/drivers/mysql/dbe.c	2016-04-04 17:02:48.983020982 +0200
@@ -43,7 +43,7 @@
 
 	if (!replace_variables(name, &datadir, &database)) {
 	    db_d_append_error(_("Unable parse MySQL embedded database name"));
-	    db_d_append_error(mysql_error(connection));
+	    db_d_append_error("%s", mysql_error(connection));
 	    db_d_report_error();
 	    return DB_FAILED;
 	}
@@ -59,7 +59,7 @@
 
 	if (mysql_server_init(4, server_args, NULL)) {
 	    db_d_append_error(_("Cannot initialize MySQL embedded server"));
-	    db_d_append_error(mysql_error(connection));
+	    db_d_append_error("%s", mysql_error(connection));
 	    db_d_report_error();
 	    free(datadir);
 	    free(database);
@@ -78,7 +78,7 @@
 
 	if (res == NULL) {
 	    db_d_append_error(_("Unable to connect to MySQL embedded server: "));
-	    db_d_append_error(mysql_error(connection));
+	    db_d_append_error("%s", mysql_error(connection));
 	    db_d_report_error();
 	    return DB_FAILED;
 	}
