--- grass-7.0.3/lib/gis/Makefile.orig	2016-04-03 19:35:16.596262959 +0200
+++ grass-7.0.3/lib/gis/Makefile	2016-04-03 21:12:20.269351895 +0200
@@ -22,7 +22,7 @@
 
 default: lib $(FMODE_OBJ) $(PROJFILES) $(COLORFILES) $(ETC)/colors.desc $(ETC)/renamed_options
 
-ifdef MINGW
+ifeq ($(MINGW),yes)
 $(FMODE_OBJ): $(OBJDIR)/fmode.o
 	$(INSTALL) $< $@
 endif
--- grass-6.4.2/lib/init/Makefile.orig	2011-06-08 12:30:46.000000000 +0200
+++ grass-6.4.2/lib/init/Makefile	2012-10-20 14:33:08.316948107 +0200
@@ -60,7 +60,7 @@
 		$(ETC)/html_browser_mac.sh
 endif
 
-ifneq ($(strip $(MINGW)),)
+ifeq ($(strip $(MINGW)),yes)
 	FILES += \
 		$(ETC)/Init.bat \
 		$(ETC)/winlocale$(EXE) \
--- grass-7.0.3/lib/python/ctypes/Makefile.orig	2016-04-03 19:35:16.626262959 +0200
+++ grass-7.0.3/lib/python/ctypes/Makefile	2016-04-03 21:13:18.706016110 +0200
@@ -58,7 +58,7 @@
 CTYPESFLAGS = --cpp "$(CC) -E $(CPPFLAGS) $(LFS_CFLAGS) $(EXTRA_CFLAGS) $(NLS_CFLAGS) $(DEFS) $(EXTRA_INC) $(INC) -D__GLIBC_HAVE_LONG_LONG"
 EXTRA_CLEAN_FILES := $(wildcard ctypesgencore/*.pyc) $(wildcard ctypesgencore/*/*.pyc)
 
-ifneq ($(MINGW),)
+ifeq ($(MINGW),yes)
 INTLLIB = -lintl-8
 endif
 
--- grass-7.0.3/gui/scripts/Makefile.orig	2016-04-03 19:35:16.652929624 +0200
+++ grass-7.0.3/gui/scripts/Makefile	2016-04-03 21:15:10.182678097 +0200
@@ -6,7 +6,7 @@
 
 DSTDIR = $(GUIDIR)/scripts
 
-ifdef MINGW
+ifeq ($(MINGW),yes)
 PYFILES := $(patsubst %.py, $(DSTDIR)/%_wrapper.py, $(filter %.py, $(PY_SOURCES)))
 BATFILES := $(patsubst %.py, $(DSTDIR)/%.bat, $(filter %.py, $(PY_SOURCES)))
 $(DSTDIR)/%_wrapper.py: %.py | $(DSTDIR)
--- grass-7.0.3/include/Make/Grass.make.orig	2014-11-03 10:57:12.000000000 +0100
+++ grass-7.0.3/include/Make/Grass.make	2016-04-03 21:16:54.589340383 +0200
@@ -80,7 +80,7 @@
 VECT_CFLAGS =  $(GDALCFLAGS) $(GEOSCFLAGS)
 
 # Object with _fmode which must be linked to each executable on Windows
-ifdef MINGW
+ifeq ($(MINGW),yes)
 WINDRES = windres
 FMODE_OBJ = $(BASE_LIBDIR)/fmode.o
 # if fmode.o causes trouble, try:
@@ -235,7 +235,7 @@
 
 ifneq ($(GDAL_LINK),)
 ifneq ($(GDAL_DYNAMIC),)
-ifneq ($(MINGW),)
+ifeq ($(MINGW),yes)
 RASTERDEPS += -lkernel32
 else
 RASTERDEPS += $(DLLIB)
@@ -277,7 +277,7 @@
 
 $(foreach lib,$(libs),$(eval $(call lib_rules,$(firstword $(subst :, ,$(lib))),$(lastword $(subst :, ,$(lib))))))
 
-ifneq ($(MINGW),)
+ifeq ($(MINGW),yes)
 GISLIB += $(INTLLIB)
 endif
 
--- grass-7.0.3/include/Make/Shlib.make.orig	2016-04-03 19:35:16.679596287 +0200
+++ grass-7.0.3/include/Make/Shlib.make	2016-04-03 21:17:48.062671474 +0200
@@ -8,7 +8,7 @@
 
 $(SHLIB): $(SHLIB_OBJS)
 	$(SHLIB_LD) -o $@ -Wl,-soname,$(notdir $@) $(LDFLAGS) $^ $(LIBES) $(EXTRA_LIBS) $(MATHLIB)
-ifndef MINGW
+ifneq ($(MINGW),yes)
 	(cd $(ARCH_LIBDIR); ln -f -s $(notdir $@) $(patsubst %.$(GRASS_VERSION_NUMBER)$(SHLIB_SUFFIX),%$(SHLIB_SUFFIX),$@))
 endif
 
--- grass-7.0.3/include/Make/Script.make.orig	2016-04-03 19:35:16.679596287 +0200
+++ grass-7.0.3/include/Make/Script.make	2016-04-03 21:18:54.369335354 +0200
@@ -1,7 +1,7 @@
 
 include $(MODULE_TOPDIR)/include/Make/Vars.make
 
-ifdef MINGW
+ifeq ($(MINGW),yes)
 SCRIPT = $(SCRIPTDIR)/$(PGM).py
 else
 SCRIPT = $(SCRIPTDIR)/$(PGM)
@@ -18,7 +18,7 @@
 include $(MODULE_TOPDIR)/include/Make/ScriptRules.make
 
 SCRIPT_ACTIONS = $(SCRIPT) $(ETCPYFILES) $(ETCPYCFILES) html scriptstrings
-ifdef MINGW
+ifeq ($(MINGW),yes)
 SCRIPT_ACTIONS += $(BIN)/$(PGM).bat
 SCRIPT_DIR = %GISBASE%/scripts
 endif
--- grass-6.4.3/include/Make/Rules.make.orig	2013-03-02 11:52:07.000000000 +0100
+++ grass-6.4.3/include/Make/Rules.make	2014-06-11 21:22:16.080021691 +0200
@@ -66,7 +66,7 @@
 %.tab.h %.tab.c: %.y
 	$(YACC) -b$* -p$* $(YACCFLAGS) $<
 
-ifneq ($(MINGW),)
+ifeq ($(MINGW),yes)
 mkpath = $(shell $(TOOLSDIR)/g.echo$(EXE) $(1));$(2)
 else
 mkpath = $(1):$(2)
--- grass-7.0.3/include/Make/GuiScript.make.orig	2014-12-17 14:10:44.000000000 +0100
+++ grass-7.0.3/include/Make/GuiScript.make	2016-04-03 21:20:29.075998049 +0200
@@ -10,7 +10,7 @@
 MODULES  := $(patsubst g.gui.%.py,%,$(wildcard g.gui.*.py))
 CMDHTML  := $(patsubst %,$(HTMLDIR)/g.gui.%.html,$(MODULES))
 GUIHTML  := $(patsubst %,$(HTMLDIR)/wxGUI.%.html,$(MODULES))
-ifdef MINGW
+ifeq ($(MINGW),yes)
 SCRIPTEXT = .py
 BATFILES  := $(patsubst %,$(BIN)/g.gui.%.bat,$(MODULES))
 else
--- grass-7.0.3/lib/proj/Makefile.orig	2015-12-17 09:13:35.000000000 +0100
+++ grass-7.0.3/lib/proj/Makefile	2016-04-03 21:20:54.752663637 +0200
@@ -36,7 +36,7 @@
 default: lib $(FTOL_OBJ)
 	$(MAKE) $(NAD_DSTFILES) $(CSV_DSTFILES) $(PROJ_DSTFILES) $(NAD_SRCFILES)
 
-ifdef MINGW
+ifeq ($(MINGW),yes)
 $(FTOL_OBJ): $(OBJDIR)/ftol.o
 	$(INSTALL) $< $@
 endif
--- grass-7.0.3/configure.in~	2015-07-19 06:50:37.000000000 +0000
+++ grass-7.0.3/configure.in	2015-07-19 06:51:21.133489471 +0000
@@ -101,10 +101,6 @@
 
 AC_SUBST(ARCH)
 
-# Use absolute path for aux directory so that install-sh works
-
-AC_CONFIG_AUX_DIR("$SRCDIR")
-
 # Set GISBASE and GRASS_BIN
 
 GISBASE="${WINDSTDIR}/dist.${ARCH}"
--- grass-7.0.3/include/Makefile.orig	2011-11-14 09:26:04.000000000 +0100
+++ grass-7.0.3/include/Makefile	2016-04-03 17:16:49.983278219 +0200
@@ -46,4 +46,4 @@
 	sed -e 's/^\(.*\)$$/"\1\\n"/' $< > $@
 
 $(ARCH_INCDIR)/confparms.h: $(MODULE_TOPDIR)/config.status | $(ARCH_INCDIR)
-	sed -n '7s/^#\(.*\)$$/"\1"/p' $< > $@
+	sed -n '7s/^#\(.*\)$$/"\1"/p;s/^ac_cs_config="\(.*"\)$$/".\/configure \1/p' $< > $@
